mlrun/api/api/endpoints/feature_sets.py

```diff
@@ -0,0 +1,88 @@
+from http import HTTPStatus
+from typing import List
+
+from fastapi import APIRouter, Depends, Response, Query
+from sqlalchemy.orm import Session
+
+from mlrun.api import schemas
+from mlrun.api.api import deps
+from mlrun.api.api.utils import log_and_raise
+from mlrun.api.utils.singletons.db import get_db
+
+router = APIRouter()
+
+
+@router.post("/projects/{project}/feature_sets")
+def add_feature_set(
+    project: str,
+    feature_set: schemas.FeatureSet,
+    versioned: bool = False,
```

Comment:
The idea here is pretty much like with functions, the feature-set may evolve during the development process. It may be less frequent changes than with functions, but the logic is the same. I'm not sure what your suggestion is - do you propose to always version feature sets?

Reply:
I was saying that I'm not sure we need it, even if we leave it, since it's already implemented, I'm pretty sure the default should be `True`

Reply:
Talked with @yaronha and he's saying it is indeed expected to happen pretty often so we're leaving as is